test/ctr.bats

```diff
@@ -778,35 +785,35 @@ function teardown() {
 		skip "travis container tests don't support testing OOM"
 	fi
 	start_crio
-	run crioctl pod run --config "$TESTDATA"/sandbox_config.json
+	run crictl runs "$TESTDATA"/sandbox_config.json
 	echo "$output"
 	[ "$status" -eq 0 ]
 	pod_id="$output"
 	oomconfig=$(cat "$TESTDATA"/container_config.json | python -c 'import json,sys;obj=json.load(sys.stdin);obj["image"]["image"] = "mrunalp/oom"; obj["linux"]["resources"]["memory_limit_in_bytes"] = 5120000; obj["command"] = ["/oom"]; json.dump(obj, sys.stdout)')
 	echo "$oomconfig" > "$TESTDIR"/container_config_oom.json
-	run crioctl ctr create --config "$TESTDIR"/container_config_oom.json --pod "$pod_id"
+	run crictl create "$pod_id" "$TESTDIR"/container_config_oom.json "$TESTDATA"/sandbox_config.json
 	echo "$output"
 	[ "$status" -eq 0 ]
 	ctr_id="$output"
-	run crioctl ctr start --id "$ctr_id"
+	run crictl start "$ctr_id"
 	echo "$output"
 	[ "$status" -eq 0 ]
 	# Wait for container to OOM
 	attempt=0
 	while [ $attempt -le 100 ]; do
 		attempt=$((attempt+1))
-		run crioctl ctr status --id "$ctr_id"
+		run crictl inspect "$ctr_id"
 		echo "$output"
 		if [[ "$output" =~ "OOMKilled" ]]; then
 			break
 		fi
 		sleep 10
 	done
 	[[ "$output" =~ "OOMKilled" ]]
```

Comment:
Maybe crictl isn't outputting the same OOMKilled string, hence failing?

Reply:
Yes, that could be likely. Could be tested manually easily.